src/structures/DMChannel.js

```diff
@@ -19,10 +19,10 @@ class DMChannel extends Channel {
     // Override the channel type so partials have a known type
     this.type = 'dm';
     /**
-     * A collection containing the messages sent to this channel
-     * @type {MessageStore<Snowflake, Message>}
+     * A manager managing the messages belonging to this channel.
```

Comment:
```suggestion
     * A manager of the messages belonging to this channel
```

"manager managing" sounds very strange, maybe "manager of"? Same applies to other instances of this